plugins/obs-qsv11/QSV_Encoder_Internal.cpp

```diff
@@ -177,6 +177,23 @@ mfxStatus QSV_Encoder_Internal::Open(qsv_param_t *pParams, enum qsv_codec codec)
 	return sts;
 }
 
+PRAGMA_WARN_PUSH
+PRAGMA_WARN_DEPRECATION
+static inline bool IsOptimizedBRCSupport(const mfxPlatform &platform,
+					 mfxU16 rateControl)
+{
+#if MFX_VERSION_MAJOR >= 2 && MFX_VERSION_MINOR >= 12
+	if (rateControl == MFX_RATECONTROL_CBR &&
```

Comment:
Does this also require a runtime version check like #11060 or is this purely a build dependency requirement?

Reply:
VPL version 2.12 adds the MFX_PLATFORM_LUNARLAKE to mfxcommon.h, which is the reason for this check.

Reply:
I suppose my question is: does OptimizedBRCSupport required VPL 2.12 on the running system as well, or is it _only_ a build-time requirement?

Reply:
It is a running requirement, but also build because the updated header needs to be on the system.

Reply:
If it's both, then this needs to also check the runtime version, like in the other PR.